HomeTask3/HomeTask3/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DevTask3;
+using System;
+
+namespace Task_Dev3
+{
+    /// <summary> 
+    /// We accept from the command line two arguments: 
+    /// 1) the number that needs to be transferred to another system of calculation 
+    /// 2) the foundation of a new system of calculus 
+    /// </summary> 
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            // We check the arguments for the correctness of the input (only numbers can be entered)
+            if(args[0] == string.Empty || args[1] == string.Empty)
+            {
+                Console.WriteLine("You entered an empty string");
+                return;
+            }
+            try 
+            {
+                long digitForTranslation;
+                long foundationOfNewsystem;
+                digitForTranslation = Int64.Parse(args[0]);
+                foundationOfNewsystem = Int64.Parse(args[1]);
+            }
```

Comment:
wrong exception usage

Reply:
Could you clarify. I don't quite understand the error,